library/src/vendored/preact-core.ts

```diff
@@ -79,9 +79,7 @@ function endBatch() {
   batchIteration = 0
   batchDepth--
 
-  if (hasError) {
-    throw internalErr(from, 'BatchError, error', { error })
-  }
+  if (hasError) throw error
```

Comment:
Is there any reason to catch the error above (line 69), rather than simply allow it to be thrown?

Reply:
cause you still need to unroll the dep before sending it to be in the right stack for the error